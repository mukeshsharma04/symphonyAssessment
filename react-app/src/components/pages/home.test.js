import React from "react";
import renderer from "react-test-renderer";
import Home from "./home";

describe("Home", () => {
  const mockProps = {
    fileUpload: { fileData: null },
    actions: { initFileUpload: jest.fn() }
  };

  test("renders correctly", () => {
    const wrapper = renderer.create(<Home {...mockProps} />).toJSON();
    expect(wrapper).toMatchSnapshot();
  });
});
