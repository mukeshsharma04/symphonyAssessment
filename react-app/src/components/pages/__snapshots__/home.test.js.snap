// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders correctly 1`] = `
<div
  className="css-83mdml"
>
  <div>
    <input
      className="css-1nvkxsy"
      onChange={[Function]}
      type="file"
    />
  </div>
  <div
    className="css-k008qs"
  >
    <div
      style={
        Object {
          "margin": "5px",
        }
      }
    >
      <label>
        Delimiter
      </label>
      <input
        className="css-1nvkxsy"
        onChange={[Function]}
        type="text"
      />
    </div>
    <div
      style={
        Object {
          "margin": "5px",
        }
      }
    >
      <label>
        Lines
      </label>
      <input
        className="css-1nvkxsy"
        onChange={[Function]}
        type="text"
      />
    </div>
  </div>
  <table
    className="css-8atqhb"
  >
    <tbody />
  </table>
</div>
`;
